config/game_options.txt

```diff
@@ -397,15 +397,15 @@ NEAR_DEATH_EXPERIENCE
 ## Set to 1 for "custom", silicons will start with the custom laws defined in silicon_laws.txt. (If silicon_laws.txt is empty, the AI will spawn with asimov and Custom boards will auto-delete.)
 ## Set to 2 for "random", silicons will start with a random lawset picked from random laws specified below.
 ## Set to 3 for "weighted random", using values in "silicon_weights.txt", a law will be selected, with weights specifed in that file.
```

Comment:
Where is silicon_weights.txt?
Haven't been able to locate that one.

If the weights are working as intended here, then this needs to be updated.

Reply:
As far as I could tell they're working.  That line was there when I got here, and I've never seen the file in question.